draft-ietf-dnsop-svcb-https.md

```diff
@@ -495,59 +495,55 @@ is the effective TargetName:
 
 # Client behavior {#client-behavior}
 
-A SVCB-aware client selects an endpoint for a service using the following
-procedure:
+"SVCB resolution" is the process of enumerating the priority-ordered endpoints
+for a service, as performed by the client.  It MUST be implemented as follows:
 
-1. Let $ADDR_QNAME be the service name.  Let $SVCB_QNAME be the service name
-   plus appropriate prefixes for the scheme (see {{svcb-names}}).
+1. Let $QNAME be the service name plus appropriate prefixes for the
+   scheme (see {{svcb-names}}).
 
-2. In parallel, issue AAAA/A queries for $ADDR_QNAME
-   and a SVCB query for $SVCB_QNAME.
-   The answers for these may or may not include CNAME pointers
```

Comment:
Do we want to keep a reference to potential of CNAMEs in-place, or this is adequately covered elsewhere and in examples?

Reply:
OK, added to Step 3.